layer.md

```diff
@@ -308,12 +308,13 @@ Any given image is likely to be composed of several of these Image Filesystem Ch
 
 # Non-Distributable Layers
 
-Certain layers, due to legal requirements, may not be regularly distributable.
-Typically, such layers are downloaded directly from a distributor but are never uploaded.
+Due to legal requirements, certain layers may not be regularly distributable.
+Such "non-distributable" layers are typically downloaded directly from a distributor but never uploaded.
 
-Layers that have these restrictions SHOULD be tagged with an alternative mediatype of `application/vnd.oci.image.layer.nondistributable.v1.tar+gzip`.
-[Descriptors](descriptor.md) referencing these layers MAY include `urls` for downloading these layers.
-It is implementation-defined whether or not implementations upload layers tagged with this media type.
+Non-distributable layers SHOULD be tagged with an alternative mediatype of `application/vnd.oci.image.layer.nondistributable.v1.tar+gzip`.
+Implementations SHOULD NOT upload layers tagged with this media type; however, such a media type SHOULD NOT affect whether an implementation downloads the layer.
+
+[Descriptors](descriptor.md) referencing non-distributable layers MAY include `urls` for downloading these layers directly.
```

Comment:
@wking More "precision" here is going to infer a hierarchy, which actually sacrifices accuracy. The term "retrieval" implies relative displaced locality. In this context, since we are talking about `urls`, the term is both accurate and precise, since the relative locality is `urls`, provided in the first clause, which implies network access. Local cache clearly does not fall into the locality level of `urls`.

Adding more here will just obscure the intent of the statement. It is neither sloppy nor imprecise and ultimately accurate in context.

But let's see what the contributor has to say, as he might have an opinion.